tests/tracking/_default_experiment/test_databricks_notebook_context.py

```diff
@@ -0,0 +1,18 @@
+from unittest import mock
+
+from mlflow.tracking._default_experiment.databricks_notebook_context import (
+    DatabricksNotebookExperimentContext,
+)
+
+
+def test_databricks_notebook_default_experiment_in_context():
+    with mock.patch("mlflow.utils.databricks_utils.is_in_databricks_notebook") as in_notebook_mock:
+        assert DatabricksNotebookExperimentContext().in_context() == in_notebook_mock.return_value
```

Comment:
Can we add a negative test case asserting that `in_context()` returns `False` by default and when `is_in_databricks_notebook()` returns `False`?

Reply:
It should work since it would return whatever the value that is returned by `in_notebook_mock`. So it could either be true or false. both tests case should be tested as far as I can tell. Let me know if I am missing something